src/utils/misc/hasPointerEvents.ts

```diff
@@ -0,0 +1,18 @@
+import {getWindowFromNode} from '@testing-library/dom/dist/helpers'
+
+export function hasPointerEvents(element: Element): boolean {
+  const window = getWindowFromNode(element)
+
+  for (
+    let el: Element | null = element;
+    el?.ownerDocument;
+    el = el.parentElement
+  ) {
+    const pointerEvents = window.getComputedStyle(el).pointerEvents
+    if (pointerEvents && ['inherit', 'unset'].includes(pointerEvents)) {
```

Comment:
Oops, dropped the `!` when adding the second exception.
```suggestion
    if (pointerEvents && !['inherit', 'unset'].includes(pointerEvents)) {
```

Those two values of `pointerEvents` don't change the behavior inherited from an ancestor. So like no value we continue traversing ancestors.
If the value is `none` we return `false`. If the value is any other value, it overrides any value of its ancestors and we return `true`.

Probably need to add a test case for this.
```html
<div style="pointer-events: none">
  <input style="pointer-events: initial"/>
  <input style="pointer-events: inherit"/>
</div>
```
The first `<input>` is clickable. The second one is not.